stacks/api/bucket/src/graphql/schema.ts

```diff
@@ -88,11 +88,15 @@ function createInterface(
 function createEnum(name: string, values: string[]) {
   return `
       enum ${name}{
-        ${values.join("\n")}
+        ${removeDuplicates(values).join("\n")}
```

Comment:
```suggestion
        ${new Set(values).join("\n")}
```

Reply:
It works with `Array.from(new Set(values).values()).join("\n")`